README.md

```diff
@@ -1,209 +1,112 @@
+[![documentation](https://img.shields.io/badge/documentation-Corbado_Backend_API_Reference-blue.svg)](https://api.corbado.com/docs/api/)
+[![Slack](https://img.shields.io/badge/slack-join%20chat-brightgreen.svg)](https://join.slack.com/t/corbado/shared_invite/zt-1b7867yz8-V~Xr~ngmSGbt7IA~g16ZsQ)
+
+The [Corbado](https://www.corbado.com) PHP SDK provides convenient access to the [Corbado Backend API](https://api.corbado.com/docs/api/) from applications written in the PHP language.
 
-## Documentation
+:warning: The Corbado Node.js SDK is commonly referred to as a private client, specifically designed for usage within closed backend applications. This particular SDK should exclusively be utilized in such environments, as it is crucial to ensure that the API secret remains strictly confidential and is never shared.
 
-For a detailed understanding of how to use the Corbado Backend API, refer to
-the [Corbado Backend API Reference](https://api.corbado.com/docs/api/)
-and [Corbado API-only integration guide](https://docs.corbado.com/integrations/api-only).
+:rocket: [Getting started](#rocket-getting-started) | :hammer_and_wrench: [Services](#hammer_and_wrench-services) | :books: [Advanced](#books-advanced) | :speech_balloon: [Support & Feedback](#speech_balloon-support--feedback)
 
-## Requirements
+## :rocket: Getting started
 
-Ensure your environment runs Node 8 or higher.
+### Requirements
+
+- Nodejs 8 or higher.
 
 ## Installation
 
 Use the following command to install the Corbado Node.js SDK:
 
-```sh
-npm install @corbado/node-sdk --save
+```bash
+npm install @corbado/node-sdk
 ```
 
 ## Usage
 
-To initialize the SDK, supply it with your Corbado account's ```project ID``` and ```API secret```. You can obtain these
-parameters
-from the [Corbado developer panel](https://app.corbado.com).
-
-## Initialization
+To create a Node.js SDK instance you need to provide your `Project ID` and `API secret` which can be found at the [Developer Panel](https://app.corbado.com).
 
 ### ES5:
 
 ```JavaScript
 const Corbado = require('@corbado/node-sdk');
 
-const projectID = process.env.PROJECT_ID;
-const apiSecret = process.env.API_SECRET;
+const projectID = process.env.CORBADO_PROJECT_ID;
+const apiSecret = process.env.CORBADO_API_SECRET;
 
-const config = new Corbado.Configuration(projectID, apiSecret);
+const config = new Corbado.Config(projectID, apiSecret);
 const corbado = new Corbado.SDK(config);
 ```
 
 ### ES6:
 
-```JavaScript 
-import {SDK, Configuration} from '@corbado/node-sdk';
+```JavaScript
+import {SDK, Config} from '@corbado/node-sdk';
 
-const projectID = process.env.PROJECT_ID;
-const apiSecret = process.env.API_SECRET;
-const config = new Configuration(projectID, apiSecret);
+const projectID = process.env.CORBADO_PROJECT_ID;
+const apiSecret = process.env.CORBADO_API_SECRET;
+const config = new Config(projectID, apiSecret);
```

Comment:
I would add a newline here as well to make this snippet similar to the one above

Reply:
Done.